go/chat/devstorage.go

```diff
@@ -16,39 +16,65 @@ type DevConversationBackedStorage struct {
 	globals.Contextified
 	utils.DebugLabeler
 
+	mt        chat1.ConversationMembersType
+	adminOnly bool
+
 	ri func() chat1.RemoteInterface
 }
 
-func NewDevConversationBackedStorage(g *globals.Context, ri func() chat1.RemoteInterface) *DevConversationBackedStorage {
+func NewDevConversationBackedStorage(g *globals.Context, mt chat1.ConversationMembersType, adminOnly bool, ri func() chat1.RemoteInterface) *DevConversationBackedStorage {
 	return &DevConversationBackedStorage{
 		Contextified: globals.NewContextified(g),
+		mt:           mt,
+		adminOnly:    adminOnly,
 		DebugLabeler: utils.NewDebugLabeler(g.GetLog(), "DevConversationBackedStorage", false),
 		ri:           ri,
 	}
 }
 
-func (s *DevConversationBackedStorage) Put(ctx context.Context, uid gregor1.UID, name string, src interface{}) (err error) {
+func (s *DevConversationBackedStorage) tlfName(ctx context.Context, tlfid chat1.TLFID) (tlfname string, err error) {
```

Comment:
This function doesn't seem right. Is there a better way to go from 

uid -> username
teamid -> teamname